components/datetime/src/format/datetime.rs

```diff
@@ -618,7 +618,7 @@ mod tests {
             .into_japanese_date()
             .to_any();
 
-        writeable::assert_writeable_eq!(dtf.format(&date).unwrap(), "Sep 1, 12 kansei-1789")
+        writeable::assert_writeable_eq!(dtf.format(&date).unwrap(), "Sep 1, 12 kansei-1789");
```

Comment:
To resolve this annoying breakage to `assert_writeable_eq`, as well as the potential that people could be using the function `writeable::writeable_to_parts_for_test`, and because we have a lot of new functionality, I think we should bump the `writeable` crate version next release.